src/pages/challenges/component.js

```diff
@@ -23,20 +24,30 @@ const UserImage = ({userImg, className}) => (
 
 export const Challenges = ({fetchChallenges, challenges, fetching}) => {
   const [tab, setTab] = useState();
+  const [search, setSearch] = useState("");
```

Comment:
I do not see where you get details of each challenge.
As I remember where need to get all challenges and then to call additional calls for each challenge